ietf-microwave-radio-link.yang

```diff
@@ -532,28 +514,16 @@ module ietf-microwave-radio-link {
             "Selected maximum coding/modulation.
              Adaptive coding/modulation shall not go
              above this value.";
+        }
+		leaf reference-modulation {
+          type identityref {
+            base mw-types:coding-modulation;
+          }
+          description
+            "reference-modulation is a reference to a coding-modulation value between the 'selected-min-acm' and the 'selected-max-acm.";
```

Comment:
Reformatted the indentation: please check use of TAB characters instead of spaces in the editor:

```suggestion
        }
        leaf reference-modulation {
          type identityref {
            base mw-types:coding-modulation;
          }
          description
            "Reference-modulation is a reference to a
             coding-modulation value between the 'selected-min-acm'
             and the 'selected-max-acm.";
```